en/middleware/micrortps_manual_code_generation.md

```diff
@@ -83,15 +64,15 @@ $ python Tools/generate_microRTPS_bridge.py -s msg/sensor_baro.msg -r msg/sensor
 
 Code is generated for the *Client*, *Agent*, *CDR serialization/deserialization* of uORB messages, and the definition of the associated RTPS messages (IDL files). 
 
-Code for the bridge can be found here:
+Code for the bridge by default in the manual generation can be found here:
```

Comment:
This path is the default for manualcreation. In the automatic generation the generated client code ends on build_PLATFORM/src/modules/micrortps_bridge/modules_client/